static/js/publisher/form/media.js

```diff
@@ -22,8 +22,13 @@ class Media extends React.Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
-    if (prevState.mediaData !== this.state.mediaData) {
-      this.props.updateState(this.state.mediaData);
+    const { mediaData } = this.state;
+    const { mediaLimit } = this.props;
+    const mediaLength = mediaData.filter(item => item.status !== "delete")
```

Comment:
No need to filter anymore (once you rebase).

Reply:
:ok_hand: